src/HitchhikersGuide.md

```diff
@@ -132,37 +132,90 @@ above.
 
 The `Orchestrator` class is used to walk all of the Actors to the next
 Phase together, and to make sure that blocking threads finish before
-other threads start their processes. It also outputs the `phaseNumber`
-to the output file using a `metrics::Gauge` object. The `Orchestrator`
-adds a token for every thread that starts, and removes the token once
-the thread is finished. When the token count is 0, it signals the end of
-the Phase.
+other threads start their processes. The `Orchestrator` adds a token for
+every thread that starts, and removes the token once the thread is
+finished. When the token count is 0, it signals the end of the Phase.
 
 ### ⏱`Metrics` ⏱
 
-Genny records metrics using the `metrics::operation` type found in `metrics.hpp`. An instance of `metrics::operation` can capture several different data points about the operation it references. 
+Genny records metrics using the `metrics::Operation` type found in
+`metrics.hpp` though the underlying implementation can be found in
+`operation.hpp`. An instance of `metrics::Operation` can capture several
+different data points about the operation it references.
 
-#### Usage: 
-In your actor class, declare a metrics::operation: 
-` metrics::Operation _operation;` and
-`_operation{context.operation("insert", Insert::id())}`
+#### Usage
 
-Begin timing the operation:
-`auto ctx = this->_operation.start();`
-(then write the code to perform the operation)
+In your actor class, declare a `metrics::Operation` member and
+initialize it using the `ActorContext::operation()` method in the
+actor's constructor.
 
-You can increment a counter (for example, increment all documents inserted in a bulkinsert):
-`ctx.addOps(1);`
+```cpp
+class MyActor {
+    MyActor(ActorContext& context)
+        : _insertOp{context.operation("Insert", MyActor::id())} {}
```

Comment:
is the order of the params right here?

Reply:
It is - when we were going over the code in-person the `v1::RegistryT::operation()` is what had `ActorId` listed first. I realized it is probably confusing for it to have them in a different order from `ActorContext::operation()` so I changed `v1::RegistryT::operation()` to have the `ActorId` listed last.